src/lib/constants/app-version.ts

```diff
@@ -0,0 +1 @@
+export const LATEST_VERSION = '1.14.10';
```

Comment:
Let me see, think we can, should we get it dynamically on the client or build?

Reply:
I'll wait for [this](https://github.com/bejamas/httptoolkit-website/pull/41) to be merged since it has some code that might be useful for getting this info.

Reply:
I added a service for this.